### ISSUE TITLE
SOCKS access for Nitheesha Karnati

### ISSUE THREAD

**Issue by NitheeshaKarnati** [department-of-veterans-affairs/va.gov-team#74202]
### Your Name

Nitheesha Karnati

### Your Email

nitheesha.karnati@va.gov, nitheesha.karnati@thoughtworks.com

### Your Role and Team

Developer

### Product Manager (PM) name and email

Nathalie.Rayter@va.gov

### Product Owner (PO) name and email

Nathalie.Rayter@va.gov

### Access Type Requesting

developers

### Public SSH Key

ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQCtm9o4kOuZgfhul4RCZDWmR4+7NzYUN7k2NJiu9TC89tYdL6jjdPPvbJfuuRMkhWaidMKw+SvQ7/+kBHtlS0C05w+ixcxskubGw2nZCXQyX8pWUbsT4DHURII4kJZ2Jk4zL2ip34H+LdL5oSh50f3NKEyQ7C+MxHeMCWzWtCVSBMx9BgtyuSkeceWN5jq51+B5fpgOP+aPUaa4LCIQGz400rzinZebkWJsBn6hpG9aKCqvlqU1WKpCWI/fMbq7ARxIbrLH2EJ+Ql8yJTNdvvdqtpva8SkLFC9BMIP2oT2GlwY5YqCKwYmKVAUNf2OZhP66FvSUuIL29ILEawM/d70Hyq5XR4xw8VVnORkW+YulFhTWM28JAAlOE9ANW2Q1S9qdQKN1i5DzR40sUOqpTAYAHGCgRXTmrFklHXzd2TeDiY+y06Yrx4wO09kKq/MexWHkc2XH7KuegPvqbCHboUmSZBbHst/B1wdY9etMkHSSGrfEduVyOd7Qs6vkUitotR1eCO0i7pkgCK3C8pW+rMAsUTB0zXo9pZaTkTZDALeot/PteuyHUiKv3/nLVbNN01FMJnLaysQoyHvS3ealzjhUekwPBpSZfBNeeNbL4Z8PHLlwu18ehC4tIiaoppV44jwc8QIEimfLjXC9DIZPrAdVrcNeUlkR/QUBCjUd3utZkQ== nitheesha.karnati@va.gov

### E-QIP Transmittal Confirmation

![Screenshot 2024-01-22 at 3 00 42 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/77509804/9b1d250c-60da-469f-afa5-e18eb713ec07)

### User must exist in a roster before SOCKS access can be granted

- [ ] Search for VFS team member in [Atlas](https://www.va.gov/atlas/product_directory/team_members)
- [ ] Or search for user on the Platform Team Roster: https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster
- [ ] If user is on a VFS team but not a team member in Atlas, add the 'NOT YET' label and instruct them to start the Platform orientation process https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html
- [ ] If a user is on a Platform team but not on the Platform Team Roster in Confluence, add the 'NOT YET' label and instruct them to reach out to their Product Manager to be added.
- [ ] Comment in this issue saying which roster the user is listed in.

**Comment by NitheeshaKarnati** [comment]
Updated the SSH key

**Comment by NitheeshaKarnati** [comment]
Hi @considerable - still waiting on SOCKS access. Could you please take a look at this?

**Comment by stoiven** [comment]
Closed and re-opened just to retrigger workflows.

**Comment by stoiven** [comment]
reached out to @NitheeshaKarnati to re-open another issue (I think this workflow on this issue is bugged)
